### PR TITLE
feat: make it possible to spawn independent `avlizer_confluent` processes

### PR DESCRIPTION
**Pull request by thalesmg**
With this, we may configure different processes with different URLs.

### CODE REVIEW

File: src/avlizer_sup.erl [hunk 1]
```diff
@@ -30,7 +30,9 @@ start_link() ->
   supervisor:start_link({local, ?SUP}, ?MODULE, []).
 
 init(_) ->
-  Children = [avlizer_confluent()],
+  %% Requires setting configs in application env
```
Comment (zmstone) [inline src/avlizer_sup.erl:33]: check app env? if not set, then don't start, otherwise start as prev

**Comment by zmstone** [comment]
please update readme.

File: src/avlizer_sup.erl (hunk 1 above)
Comment (thalesmg) [reply src/avlizer_sup.erl:33]: :checkered_flag:

**Comment by thalesmg** [comment]
> please update readme.

:checkered_flag:

**Review by zmstone — APPROVED** [review]

**Comment by thalesmg** [comment]
tagged 0.5.1.1

**Pull request merged**